bin/MessagesBundle_en.properties

```diff
@@ -53,14 +53,14 @@ Circ_opt = Options
 Macro_origin = Show macro origin
 View = View
 Export_completed = Export has been done.
-Export_error = Error during exporting: 
+Export_error = Error during exporting:
```

Comment:
The spaces play a role, here for the spacing. I would not remove them, unless you solved the problem of the spacing in the layout of the dialog.